scipy/stats/tests/test_distributions.py

```diff
@@ -4318,28 +4318,14 @@ def test_boost_eval_issue_14606(self):
             stats.beta.ppf(q, a, b)
 
     @pytest.mark.parametrize('method', [stats.beta.ppf, stats.beta.isf])
-    @pytest.mark.parametrize('a, b', [(1e-310, 12.5), (12.5, 1e-310)])
+    @pytest.mark.parametrize('a, b', [(1e-323, 5), (5, 1e-323),
+                                      (1e-310, 12.5), (12.5, 1e-310)])
     def test_beta_ppf_with_subnormal_a_b(self, method, a, b):
-        # Regression test for gh-17444: beta.ppf(p, a, b) and beta.isf(p, a, b)
-        # would result in a segmentation fault if either a or b was subnormal.
+        # Regression test for gh-17444 and gh-18906: beta.ppf(p, a, b) and
+        # beta.isf(p, a, b) would result in a segmentation fault if either a
+        # or b was subnormal.
         p = 0.9
-        # Depending on the version of Boost that we have vendored and
-        # our setting of the Boost double promotion policy, the call
-        # `stats.beta.ppf(p, a, b)` might raise an OverflowError or
-        # return a value.  We'll accept either behavior (and not care about
```

Comment:
This comment does not indicate conclusively that the `OverflowError` is going to go away. I'll note that it seems to depend on the double promotion policy, and we do have:
```
-DBOOST_MATH_PROMOTE_DOUBLE_POLICY=false'
```
which we don't want to change I think.

Reply:
This part of the comment does not. Did you see the rest of the comment?

Reply:
Yes, and I read the linked issue and PRs. It's ambiguous whether this is fixed; issue 882 is still open and the denormal handling in the PRs that link it are for `ibeta` not `beta`.

Reply:
Ah I interpreted the comment differently. I thought "1.80 or earlier" meant it was fixed in 1.81. 
I think `ibeta` is the incomplete beta function, which is the CDF of the beta distribution. So the inverse of that is the ppf. That said, our `beta` distribution doesn't use Boost's `ibeta_inverse` directly, but rely on its `beta` distribution, and perhaps that is different and wasn't fixed.

Reply:
There was also some related work done on https://github.com/boostorg/math/issues/1006. The fix to that issue resolved a kernel crash locally, but it seems that it didn't fix remaining `OverflowError`s.